blackjax/mcmc/metrics.py

```diff
@@ -64,6 +65,7 @@ class Metric(NamedTuple):
     sample_momentum: Callable[[PRNGKey, ArrayLikeTree], ArrayLikeTree]
     kinetic_energy: KineticEnergy
     check_turning: CheckTurning
+    scale: Callable[[ArrayLikeTree, ArrayLikeTree, bool], ArrayLikeTree]
```

Comment:
turn in to a Protocal as well? That way we can see also the names

Reply:
Ok